hypothesis-python/tests/cover/test_annotations.py

```diff
@@ -84,7 +84,7 @@ def f(*, a, b=2):
 
 
 def pointless_composite(draw: None, strat: bool, nothing: list) -> int:
```

Comment:
Let's rename this, since it's no longer pointless.  Maybe `to_wrap_with_composite`?